plugins/chat/db/post_migrate/20230627045111_set_last_viewed_at_for_user_chat_channel_memberships.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class SetLastViewedAtForUserChatChannelMemberships < ActiveRecord::Migration[7.0]
+  def up
+    DB.exec("UPDATE user_chat_channel_memberships SET last_viewed_at = NOW()")
```

Comment:
We don't really have anything meaningful to set this to, over time it will become more accurate.

Reply:
Shouldn't this be the default maybe and enforce it to never be null? That way we don't have to deal with `last_viewed_at` being null?

Reply:
Yeah good point will add a default :+1: